whitenoise/string_utils.py

```diff
@@ -1,12 +1,7 @@
 import sys
 
-
-if sys.version_info[0] >= 3:
-    BINARY_TYPE = bytes
-    TEXT_TYPE = str
-else:
-    BINARY_TYPE = str
-    TEXT_TYPE = unicode  # noqa: F821
+BINARY_TYPE = bytes
+TEXT_TYPE = str
```

Comment:
These constants are redundant now and references to them should be just be replaced with `bytes` and `str`

Reply:
Done